setup.py

```diff
@@ -1,23 +1,28 @@
 import os
 import sys
-sys.path.append('nylas/')
-
+import re
 from setuptools import setup, find_packages
 from setuptools.command.test import test as TestCommand
-from _client_sdk_version import __VERSION__
+
+
+VERSION = ''
+with open('nylas/_client_sdk_version.py', 'r') as fd:
+    VERSION = re.search(r'^__VERSION__\s*=\s*[\'"]([^\'"]*)[\'"]',
+                        fd.read(), re.MULTILINE).group(1)
```

Comment:
This code allows us to avoid modifying `sys.path`, which is generally an anti-pattern. You can verify that the setup.py file still gets the same version number by running `python setup.py --version`.